### PR TITLE
New page at /rental-property-location

### PR DESCRIPTION
**Pull request by JoeChapman**

### CODE REVIEW

File: apps/right-to-rent-check/acceptance/features/rental-property-location.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+});
+
+Scenario('Then the correct fields are on the page', (
+  I
+) => {
+  I.seeElement('#rental-property-location-england');
+  I.seeElement('#rental-property-location-scotland');
+  I.seeElement('#rental-property-location-wales');
+  I.seeElement('#rental-property-location-ireland');
+});
```
Comment (sulthan-ahmed) [inline apps/right-to-rent-check/acceptance/features/rental-property-location.js]: looks fine but have you considered? 
```
I.seeElements([
 '#rental-property-location-england',
'#rental-property-location-scotland'...
])
```

**Comment by JoeChapman** [comment]
@sulthan-ahmed do you foresee  problem here?

File: apps/right-to-rent-check/acceptance/features/rental-property-location.js (hunk 1 above)
Comment (JoeChapman) [reply apps/right-to-rent-check/acceptance/features/rental-property-location.js]: no I haven't

**Comment by sulthan-ahmed** [comment]
I don't foresee a problem.  I think it looks good, just a minor suggested change.

**Pull request merged**